config/initializers/black_light_silence.rb

```diff
@@ -0,0 +1,8 @@
+# Avalon version five writes a lot of deprecation warnings to
+# stderr (which goes to the apache error log). These warnings
+# are fixed in Avalon version six. 
+if Avalon::VERSION.to_i < 6
+  if [:production, :staging, :test].include?(Rails.env.to_sym)
+    Deprecation.default_deprecation_behavior = :silence
```

Comment:
Sucks you have to do this, I am assuming avalon is very similar to sufia where they didn't provide alternatives and a upgrade path to remove these deprecation warnings?

Also I was going to say shouldn't this be ActiveSupport::Deprecation but looks like Hydra created their own Deprecation gem or something weird haha :/